features/bo_new/dashboard/manage_users.feature

```diff
@@ -0,0 +1,38 @@
+@bo_new @upper_tier @bo_dashboard
+Feature: [RUBY-759] Manage users
+  As a super user
+  I want to amend EA user privileges
+  So that the right people can help our users
+
+Scenario: Agency super user adds an agency user then upgrades them
+  Given I sign into the back office as "agency-super"
+  When I invite a new "agency" user
+  Then the new user has the correct back office permissions
+  When the new user accepts its invitation and setup a password
+  Then the new user is logged in
+  When I sign into the back office as "agency-super"
+    And I update the new user role to an "agency with refund"
+  Then the new user has the correct back office permissions
+
+Scenario: Agency super user adds and removes an agency user
+  Given I sign into the back office as "agency-super"
+    And I invite a new "agency" user
+    And the new user accepts its invitation and setup a password
```

Comment:
Can we amend to `the new user accepts their invitation and sets up a password`?

Reply:
Definitely!! :D My broken English :P